packages/clients/src/api/.eslintrc

```diff
@@ -5,6 +5,7 @@
     "@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "defaults|request" }],
     "max-classes-per-file": "off",
     "tsdoc/syntax": "off",
-    "no-underscore-dangle": ["error", { "allowAfterThis": true }]
+    "no-underscore-dangle": ["error", { "allowAfterThis": true }],
+    "deprecation/deprecation": "off"
```

Comment:
We only want it for generated api no ?

Reply:
Yes, this `eslintrc` is on the adequate subfolder ^^

But we could move this file to the root file, and constraint to the path instead.

Reply:
Oh yeah right, didn't pay attention to the path, my bad :)